### PR TITLE
Rust 2.0

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

File: demo/rust/micdemo/src/main.rs [hunk 1]
```diff
@@ -13,20 +13,21 @@ use chrono::prelude::*;
...
-use rhino::{RhinoBuilder};
 use pv_recorder::{Recorder, RecorderBuilder};
+use rhino::RhinoBuilder;
 use std::sync::atomic::{AtomicBool, Ordering};
 
 static LISTENING: AtomicBool = AtomicBool::new(false);
 
 fn rhino_demo(
     audio_device_index: i32,
+    access_key: &str,
```
Comment (laves) [inline demo/rust/micdemo/src/main.rs:24]: add `require_endpoint` as a command-line arg

File: demo/rust/filedemo/src/main.rs [hunk 2]
```diff
@@ -18,11 +18,12 @@ use std::path::PathBuf;
 
 fn rhino_demo(
     input_audio_path: PathBuf,
+    access_key: &str,
```
Comment (laves) [inline demo/rust/filedemo/src/main.rs:21]: add `require_endpoint` as a command-line arg

File: binding/rust/Cargo.toml [hunk 3]
```diff
@@ -33,6 +33,7 @@ crate_type = ["lib"]
 libc = "0.2"
 libloading = "0.7"
 log = "0.4"
+lazy_static = "1.4.0"
```
Comment (laves) [inline binding/rust/Cargo.toml:36]: update rhino version to 2.0.0

**Review by laves — CHANGES_REQUESTED** [review]
I noticed when I gave the file demo the `test_out_of_context.wav` file it gave me:
```console
Did not understand the command
Did not understand the command
Did not understand the command
```
The demo should just exit the process loop as soon as you `is_finalized == true`

**Comment by ErisMik** [comment]
> I noticed when I gave the file demo the `test_out_of_context.wav` file it gave me:
> 
> ```
> Did not understand the command
> Did not understand the command
> Did not understand the command
> ```
> 
> The demo should just exit the process loop as soon as you `is_finalized == true`

Do we want this behaviour? If I have an audio file with multiple intents in it should it not try and detect all of them?

**Comment by laves** [comment]
The other Rhino demos all just do a single intent, so for consistency's sake let's keep it to one. You would need to call the reset routine in the loop to do multiple.

**Review by laves — APPROVED** [review]

**Pull request merged**